tests/flakes/flakes.sh

```diff
@@ -468,3 +478,7 @@ nix store delete $(nix store add-path $badFlakeDir)
 
 [[ $(nix path-info      $(nix store add-path $flake1Dir)) =~ flake1 ]]
 [[ $(nix path-info path:$(nix store add-path $flake1Dir)) =~ simple ]]
+
+# Test shebang
+[[ $($nonFlakeDir/shebang.sh) = "foo" ]]
```

Comment:
Arguments must be passed to the script (and the script should echo it back to make the test work)
Something like:

```suggestion
[[ $($nonFlakeDir/shebang.sh "foo bar") = "arg[1] was foo bar" ]]
```

Reply:
Yes, args are passed along in the expected way. Updated the test. Thanks.